testProjectKotlinDslCopySpec/build.gradle.kts

```diff
@@ -0,0 +1,59 @@
+import com.google.protobuf.gradle.generateProtoTasks
+import com.google.protobuf.gradle.id
+import com.google.protobuf.gradle.ofSourceSet
+import com.google.protobuf.gradle.plugins
+import com.google.protobuf.gradle.protobuf
+import com.google.protobuf.gradle.protoc
+
+plugins {
+  java
+  id("java-library")
+  id("com.google.protobuf").version("0.8.15")
```

Comment:
This doesn't seem right. You shouldn't specify the 0.8.15 version. Otherwise, it's not actually testing the plugin that built from the project (tests still pass, which means the change isn't really get covered. Or the Gradle TestKit just ignores the version while still injecting the plugin built from code to the classpath of the test build? I am not sure, try delete this version setting and confirm you are really testing the code instead of the plugin pulled from Maven).

Reply:
Thanks for spotting.  This is an interesting one, I will investigate it.

Reply:
I have pushed a sanity check to confirm that the test fails under 0.8.15 - pushed here:
https://github.com/tom-haines/protobuf-gradle-plugin/tree/example-project/prove_test_fail